src/lib/ElementDescriptorContainer.js

```diff
@@ -61,7 +62,6 @@ import BezierCurveToDescriptor from './descriptors/Geometry/Shapes/BezierCurveTo
 import QuadraticCurveToDescriptor from './descriptors/Geometry/Shapes/QuadraticCurveToDescriptor';
 import AbsArcDescriptor from './descriptors/Geometry/Shapes/AbsArcDescriptor';
 import AbsEllipseDescriptor from './descriptors/Geometry/Shapes/AbsEllipseDescriptor';
-import HoleDescriptor from './descriptors/Geometry/Shapes/HoleDescriptor';
```

Comment:
ok you're correct looks like it wasn't commented as I thought I might have accidently done that localy. I will restore